src/Mantasflowers.Contracts/Order/Response/GetDetailedOrderResponse.cs

```diff
@@ -1,10 +1,11 @@
-﻿using Mantasflowers.Domain.Enums;
+﻿using Mantasflowers.Contracts.Common;
+using Mantasflowers.Domain.Enums;
 using System;
 using System.Collections.Generic;
 
 namespace Mantasflowers.Contracts.Order.Response
 {
-    public class GetDetailedOrderResponse
+    public class GetDetailedOrderResponse : VersionableDtoTemplate
```

Comment:
`Etag` header is sent over in the header. In fact, this wouldn't even work, because `RowVersion` has `[JsonIgnore]`